ocrd_tesserocr/common.py

```diff
@@ -173,10 +173,9 @@ def image_from_page(workspace, page, page_id):
                  'y': 0,
                  'w': page_image.width,
                  'h': page_image.height}
-    # FIXME: uncomment as soon as we get @orientation in PageType:
-    # # region angle: PAGE orientation is defined clockwise,
-    # # whereas PIL/ndimage rotation is in mathematical direction:
-    # page_xywh['angle'] = -(page.get_orientation() or 0)
+    # region angle: PAGE orientation is defined clockwise,
+    # whereas PIL/ndimage rotation is in mathematical direction:
+    page_xywh['angle'] = -(page.get_orientation() or 0)
     # FIXME: remove PrintSpace here as soon as GT abides by the PAGE standard:
```

Comment:
@tboenig @VolkerHartmann just a reminder to revisit once you're ready with the updated GT